shell/packages/sandstorm-db/db.js

```diff
@@ -938,6 +974,21 @@ _.extend(SandstormDb.prototype, {
 
 if (Meteor.isServer) {
   SandstormDb.prototype.getWildcardOrigin = getWildcardOrigin;
+
+  const Crypto = Npm.require("crypto");
+  SandstormDb.prototype.removeApiTokens = function (query) {
```

Comment:
Ehh.... In theory I like that, but in practice I worry that it will have some unintended consequences, and I'm not excited about thoroughly testing it right now. Also note that some of our old migration use ApiTokens.remove, and I didn't update them -- I _could_ update them but then I'd need to test them...